command/results_gotest_parser.go

```diff
@@ -160,16 +155,17 @@ func (vp *goTestParser) handleEnd(line string, rgx *regexp.Regexp) error {
 	if err != nil {
 		return errors.Wrapf(err, "error parsing end line '%s'", line)
 	}
-	t := vp.tests[name]
-	if t == nil || t.Name == "" {
+	tAry := vp.tests[name]
```

Comment:
should we check the ok value for this purpose? 

I'm a touch suspicious of slices of pointers, by default, but that wasn't your decision, and it might be right about this, but I haven't thought enough about it.

Reply:
Added checking